upgrade/lab1.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+echo "
+OBJECTIVE
+---------
+
+Perform a manual upgrade of the installed cluster.
+
+
+INSTRUCTIONS
+------------
+
+From the upgrade tarball, upload the new version to the cluster:
+
+    sudo ./upload
+
+Trigger the operation in manual mode:
+
+    sudo ./gravity upgrade --manual
+
+Inspect the operation plan and step through the manual upgrade until completion.
+
+After successful upgrade, uninstall the cluster by running the following command on all three of the nodes:
+
+    sudo gravity system uninstall --confirm
+
+Then reset the cluster to the original 5.5.43 state from lab0.
```

Comment:
Did we decide to go with `5.5.43` or will we be using the latest `5.5.X` version (probably `5.5.50`). Maybe just keep it general?

Reply:
Fair enough that makes sense to me